### PR TITLE
Graph task tests

### PR DESCRIPTION
**Pull request by dbart01**

### CODE REVIEW

File: BuyTests/Graph.Task.swift [hunk 1]
```diff
@@ -0,0 +1,97 @@
+//
+//  Graph.Task.swift
```
Comment (bgulanowski) [inline BuyTests/Graph.Task.swift]: I don't understand this naming convention.

File: BuyTests/Graph.Task.swift [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import XCTest
+@testable import Buy
+
+class Graph_Task: XCTestCase {
```
Comment (bgulanowski) [inline BuyTests/Graph.Task.swift]: What's wrong with `TaskTest`? "Graph" seems verbose. It's implicit.

File: BuyTests/Graph.Task.swift (hunk 1 above)
Comment (dbart01) [inline BuyTests/Graph.Task.swift]: `Task` is a nested class under the `Graph` namespace. It neatly groups all classes pertaining to GraphQL under one namespace.

File: BuyTests/Graph.Task.swift (hunk 2 above)
Comment (dbart01) [inline BuyTests/Graph.Task.swift]: This name is incorrect and is fixed in future PRs.

**Pull request merged**